pkg/cloud/aws/services/ec2/ami.go

```diff
@@ -16,43 +16,53 @@ limitations under the License.
 
 package ec2
 
+import (
+	"fmt"
+	"strings"
+
+	"k8s.io/klog"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/pkg/errors"
+)
+
+func amiName(platform, platformVersion, kubernetesVersion string) string {
```

Comment:
I think we should probably look at different naming here rather than `platform` and `platformVersion` as it overloads the use of `platform` in Go

Reply:
This method should also probably take the container runtime as an argument as well, I suspect there will be a use case in the future to have alternative container runtimes.